__tests__/auth-routes.test.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+process.env.SECRET = "toes";
+
+const server = require('')
```

Comment:
Everything looks really great, I would just like to see tests for the v1 or v2 routes